app/assets/stylesheets/projects.scss

```diff
@@ -657,7 +657,7 @@ li.module-hover {
 
 // New projects page
 .projects-container {
-  margin-top: 35px;
+  margin: 35px 20px;
```

Comment:
Why you need margin-bottom?

Reply:
Because there is 0 margin now and it doesn't look good.